gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/configuration/dictionary/DictionaryService.java

```diff
@@ -32,7 +32,7 @@ public interface DictionaryService {
     DictionaryEntity update(ExecutionContext executionContext, String id, UpdateDictionaryEntity dictionary);
     DictionaryEntity updateProperties(ExecutionContext executionContext, String id, Map<String, String> properties);
 
-    DictionaryEntity findById(String id);
+    DictionaryEntity findById(ExecutionContext executionContext, String id);
```

Comment:
environmentId ?

Reply:
I prefer keeping execution context, in case we decide to attach dictionaries to an organization